src/Pim/Component/Catalog/Factory/ProductValue/MetricProductValueFactory.php

```diff
@@ -48,7 +48,7 @@ public function create(AttributeInterface $attribute, $channelCode, $localeCode,
         if (null === $data) {
             $data = [
                 'amount' => null,
-                'unit'   => $attribute->getDefaultMetricUnit(),
```

Comment:
To be sure this will not result in an API BC breack. But you will have fails on Integration tests and/or behats for sure.